### ISSUE TITLE
x/pkgsite: Invalid license Artistic-1.0-Perl

### ISSUE THREAD

**Issue by kivilahtio** [golang/go#45441]
### What is the URL of the page with the issue?
 https://pkg.go.dev/github.com/kivilahtio/go-re@v0.1.0/v0
 https://github.com/kivilahtio/go-re
 
 
### What is your user agent?
 Ubuntu 20.04
 Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:87.0) Gecko/20100101 Firefox/87.0
 
### What did you do?
 According to your license policy:
     https://pkg.go.dev/license-policy
 I implemented a release test to check that the license is free to distribute using the same library you do:
     https://pkg.go.dev/github.com/google/licensecheck
     https://github.com/kivilahtio/go-re/blob/master/xt/LICENSE_test.go
 
### What did you expect to see?
 I expected the following license to be recognizable as a distributable license:
     Artistic-1.0-Perl

Could you add the Artistic-1.0-Perl license as a freely distributable one?
I chose the Perl-variant specifically because the library uses strong Perl idioms.
Basically it is the artistic license with an acknowledgement to previous generations' of work.

**Comment by jba** [comment]
I'll ask if we can accept it.

https://opensource.org/licenses/Artistic-Perl-1.0 says that it has been superseded by v2. Any reason not to use v2?

**Comment by jba** [comment]
We already accept Artistic-2.0 so it seems like the best course of action is to use the non-deprecated version of the license. It doesn't seem good for the community to proliferate obsolete licenses.

**Comment by kivilahtio** [comment]
Hi!
Thanks for investigating. I switched to using Artistic-2.0
